Build problem on Alpine linux 3.21

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks! -->

**Describe the bug**

<!-- A clear and concise description of what the bug is. -->
Falco binary cannot build on Alpine linux 3.21/3.22
**How to reproduce it**

<!-- Minimal and precise steps to reproduce the bug. -->
```
docker run -it alpine:3.22
apk add g++ gcc cmake make git bash perl linux-headers autoconf automake m4 libtool elfutils-dev libelf-static binutils bpftool clang
git clone https://github.com/falcosecurity/falco.git -b 0.41.0
cd falco
mkdir -p build
cd build
cmake -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_DRIVER=OFF -DCPACK_GENERATOR=TGZ -DBUILD_BPF=OFF -DBUILD_FALCO_MODERN_BPF=ON -DBUILD_WARNINGS_AS_ERRORS=OFF -DFALCO_VERSION=0.41.0 -DUSE_BUNDLED_DEPS=ON /src/falco
make package
```

Error:
```
[  1%] Building CXX object third_party/abseil-cpp/absl/base/CMakeFiles/absl_malloc_internal.dir/internal/low_level_alloc.cc.o
In file included from /src/falco/build/grpc-prefix/src/grpc/third_party/abseil-cpp/absl/base/internal/low_level_alloc.cc:26:
/src/falco/build/grpc-prefix/src/grpc/third_party/abseil-cpp/absl/base/internal/direct_mmap.h:75:25: error: 'off64_t' has not been declared
   75 |                         off64_t offset) noexcept {
      |                         ^~~~~~~
make[5]: *** [third_party/abseil-cpp/absl/base/CMakeFiles/absl_malloc_internal.dir/build.make:79: third_party/abseil-cpp/absl/base/CMakeFiles/absl_malloc_internal.dir/internal/low_level_alloc.cc.o] Error 1
make[4]: *** [CMakeFiles/Makefile2:1986: third_party/abseil-cpp/absl/base/CMakeFiles/absl_malloc_internal.dir/all] Error 2
make[3]: *** [Makefile:136: all] Error 2
make[2]: *** [CMakeFiles/grpc.dir/build.make:86: grpc-prefix/src/grpc-stamp/grpc-build] Error 2
make[1]: *** [CMakeFiles/Makefile2:1054: CMakeFiles/grpc.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
```
**Expected behaviour**

Succesful build.
<!-- A clear and concise description of what you expected to happen. -->

**Screenshots**


<!-- If applicable, add screenshots to help explain your problem. -->

**Environment**

<!-- Please complete the following info. -->

- Falco version: 0.41.0
<!-- Use "falco --version". -->
- System info:
<!-- Falco has a built-in support command you can use  "falco --support | jq .system_info" -->
- Cloud provider or hardware configuration:
- OS: Alpine linux 3.22
<!-- Eg., output of "cat /etc/os-release". -->
- Kernel:
<!-- Eg., output of "uname -a". -->
- Installation method:
<!-- Eg., Kubernetes, RPM, DEB, from source? -->

**Additional context**

<!-- Add any other context about the problem here. -->

--- a/cmake/modules/grpc.cmake
+++ b/cmake/modules/grpc.cmake
@@ -181,6 +181,10 @@ else()
 			sh
 			-c
 			"sed -i '20s/^/#include <cstdint>/' ${GRPC_SRC}/third_party/abseil-cpp/absl/strings/internal/str_format/extension.h"
+			&&
+			sh
+			-c
+			"sed -i 's|off64_t|off_t|g' ${GRPC_SRC}/third_party/abseil-cpp/absl/base/internal/direct_mmap.h"
 		)
 
 		# Zig workaround: Add a PATCH_COMMAND to grpc cmake to fixup emitted -march by abseil-cpp

